internal/categories/types.go

```diff
@@ -30,7 +30,7 @@ func FromDBType(dbCategory *db.Category) *Category {
 }
 
 func FromDBTypeArray(dbCategories []*db.Category) []*Category {
-	var categories []*Category
+	categories := []*Category{}
```

Comment:
This is just to set this to be an empty slice rather than nil for purpose of API response